packages/stats/gbstats/frequentist/tests.py

```diff
@@ -85,7 +85,9 @@ def frequentist_variance_relative_cuped(
     )
     v_trt = num_trt / den_trt
     const = -stat_b.post_statistic.mean
-    num_a = stat_a.post_statistic.variance * const**2 / (stat_a.post_statistic.mean**2)
+    num_a = (
+        stat_a.post_statistic.variance * const**2 / (stat_a.post_statistic.mean**2)
+    )
```

Comment:
hmm not sure how these files got formatted and added when I'm prettty sure I never opened this, but can't hurt to have them prettified.